common/vospi.c

```diff
@@ -196,6 +196,11 @@ int vospi_deinit() {
     return omv_spi_deinit(&vospi.spi_bus);
 }
 
+int vospi_abort(void) {
+    vospi.flags &= ~VOSPI_FLAGS_CAPTURE;
+    return omv_spi_transfer_abort(&vospi.spi_bus);
```

Comment:
vospi.flags &= ~VOSPI_FLAGS_CAPTURE;
vospi.pid = 0;
vospi.sid = 0;
omv_spi_transfer_abort(&vospi.spi_bus);
vospi.flags |= VOSPI_FLAGS_RESYNC;

Reply:
pid/sid need to be zeroed and rsync set to startup the vospi interface again if you do something like set_pixformat() -> snapshot() ->set_pixformat() -> snapshot().

Reply:
Okay, I refactor all the code that does similar abort.